v5_account_service_test.go

```diff
@@ -28,23 +28,27 @@ func TestV5Account_GetWalletBalance(t *testing.T) {
 						"accountMMRate":          "0",
 						"totalPerpUPL":           "-0.11001349",
 						"totalWalletBalance":     "18070.43799271",
+						"accountLTV":             "0",
 						"totalMaintenanceMargin": "0.38106773",
 						"coin": []map[string]interface{}{
 							{
 								"availableToBorrow":   "2.5",
+								"bonus":               "0",
 								"accruedInterest":     "0",
 								"availableToWithdraw": "0.805994",
 								"totalOrderIM":        "0",
 								"equity":              "0.805994",
 								"totalPositionMM":     "0",
 								"usdValue":            "12920.95352538",
 								"unrealisedPnl":       "0",
+								"collateralSwitch":    true,
+								"spotHedgingQty":      "0",
 								"borrowAmount":        "0",
 								"totalPositionIM":     "0",
 								"walletBalance":       "0.805994",
-								"free":                "",
```

Comment:
📝 for https://github.com/hirokisan/bybit/pull/187/files#r1814580213, this change is not necessary

Reply:
💭 I see, so for some reason it is not included in the response returned from the server.